lib/util/lifted/influx/coordinator/statement_executor.go

```diff
@@ -1087,6 +1087,15 @@ func (e *StatementExecutor) executeSelectStatement(stmt *influxql.SelectStatemen
 				closed = true
 				break
 			}
+			if stmt.Location != nil {
```

Comment:
![image](https://github.com/openGemini/openGemini/assets/31127693/88eea29a-7f90-4e71-afd8-6e9dbdcfb72f)
httpsendtransform has implemented setLoc.() Please check whether the current modification can pass or whether the client does not support the timestamp format switching. Maybe you can run the -help command to check the commands supported by the client.

Reply:
hi @lihanxue , I'm not familiar enough with openGemini yet and will need more context to fully understand your comment.

But according to my current understanding, I can confirm that `ts-cli` cannot output correct timestamp when timezone is used, and the issue could be fixed by this PR.

Reply:
You can verify the problem based on this  PR. If it doesn't work then it should be a problem of ts-cli.

Reply:
I'm not sure what `verify the problem based on this PR` mean, I have tried using `ts-cli` with this PR and I can confirm it works, or I will not submit this PR.

Reply:
Sorry, I always use the internal source version or open souce main for verification. This issue has been resolved later than open source v1.2.0. You can use main for verification.

Reply:
commit and code:
![image](https://github.com/openGemini/openGemini/assets/31127693/65a8834f-6b54-4180-be35-2ff1a8e7b1f3)
![image](https://github.com/openGemini/openGemini/assets/31127693/23609879-e78e-43aa-ab11-b05d6cf97f5e)